explorer/src/Pos/Explorer/Web/ClientTypes.hs

```diff
@@ -357,6 +364,12 @@ data CGenesisAddressInfo = CGenesisAddressInfo
     , cgaiIsRedeemed     :: !Bool
     } deriving (Show, Generic)
 
+data CAddressesFilter =
+      RedeemedAddresses
```

Comment:
`=` aligned with `|`?

Reply:
See definition of `CAddressType` in the same file. We're already using this formatting convention.